src/index.js

```diff
@@ -0,0 +1,7 @@
+import {
```

Comment:
Can simplify to `export { /* ...all the imports here */ } from './Grid'`. Or, if `Grid.js` gets split up, this file can `export { default as Column } from './Column';`, etc.